src/Octoshift/Services/ConfirmationService.cs

```diff
@@ -3,61 +3,61 @@ namespace OctoshiftCLI.Services
 {
     public class ConfirmationService
     {
-        # region Variables
-
-        private readonly Action<string> _writeToConsoleOut;
+        private readonly Action<string, ConsoleColor> _writeToConsoleOut;
         private readonly Func<ConsoleKey> _readConsoleKey;
+        private readonly Action<int> _cancelCommand;
 
-        #endregion
-
-        #region Constructors
         public ConfirmationService()
         {
-            _writeToConsoleOut = msg => Console.WriteLine(msg);
+            _writeToConsoleOut = (msg, outputColor) =>
+            {
+                Console.ForegroundColor = outputColor;
+                Console.WriteLine(msg);
+            };
             _readConsoleKey = ReadKey;
+            _cancelCommand = code => Environment.Exit(code);
         }
 
         // Constructor designed to allow for testing console methods
-        public ConfirmationService(Action<string> writeToConsoleOut, Func<ConsoleKey> readConsoleKey)
+        public ConfirmationService(Action<string, ConsoleColor> writeToConsoleOut, Func<ConsoleKey> readConsoleKey, Action<int> cancelCommand)
         {
             _writeToConsoleOut = writeToConsoleOut;
             _readConsoleKey = readConsoleKey;
+            _cancelCommand = cancelCommand;
         }
 
-        #endregion
-
-        #region Functions
-        public bool AskForConfirmation(string confirmationPrompt, string cancellationErrorMessage = "")
+        public virtual bool AskForConfirmation(string confirmationPrompt, string cancellationErrorMessage = "")
         {
             ConsoleKey response;
             do
             {
-                _writeToConsoleOut(confirmationPrompt);
+                _writeToConsoleOut(confirmationPrompt, ConsoleColor.Yellow);
+                Console.ForegroundColor = ConsoleColor.White;
                 response = _readConsoleKey();
                 if (response != ConsoleKey.Enter)
                 {
-                    _writeToConsoleOut("");
```

Comment:
Same as the other default value comment... I think it would be a great once this is implemented though!

"This doesn't seem to be a c# feature (yet): https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/lambda-method-group-defaults#relevant-background

Unless we want to use preview but I don't think it's worth using an unsupported feature."